### DISCUSSION TITLE
Can IP whitelist be shared and used?

### DISCUSSION THREAD

**Discussion by iGeeky** [apache/apisix#5772] [Q&A]
Hi all, I am using apisix in my company project, and now I have a question about using it:

I have 5-10 routes, all of them need to use IP whitelist, and they need to use the same IP whitelist, 
Is there any way to share a set of whitelists?
Because I have to add IPs often, I do not want to add each route once.
Also can not be added globally, because there are other routes are not needed or use other whitelists.

**Comment by tokers** [comment]
See https://apisix.apache.org/docs/apisix/architecture-design/service/.

**Comment by iGeeky** [comment]
I read the documentation, the service needs to be the same service on the backend to work.
This should not apply to my case. Because these routes are not the same domain name and backend service.

**Reply by tokers** [reply]
You can just put this plugin to the service but still, preserving all the discrepant parts to the route level.

**Reply by iGeeky** [reply]
Now 5 routes are 5 different services. How to set up the service to use the same IP whitelist?
